src/components/PageBlocks/Report/Configurator.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <b-tab :title="$t('report.label')">
+    <b-row>
+      <b-col>
+        <b-form-group
+          :label="$t('report.label')"
+        >
+          <b-form-select
+            v-model="options.reportID"
+            :options="reportOptions"
+            text-field="handle"
+            value-field="reportID"
+          />
+        </b-form-group>
+      </b-col>
+      <b-col
+        v-if="selectedReport && scenarioOptions.length > 1"
+      >
+        <b-form-group
+          :label="$t('report.scenario.label')"
+        >
+          <b-form-select
+            v-model="options.scenarioID"
+            :options="scenarioOptions"
+            text-field="label"
+            value-field="key"
+          />
+        </b-form-group>
+      </b-col>
+    </b-row>
+
+    <b-row>
+      <b-col>
+        <b-form-group
+          v-if="selectedReport"
+          :label="$t('report.block.label')"
+        >
+          <b-form-select
+            v-model="options.blockID"
+            :options="blockOptions"
+            text-field="title"
+            value-field="key"
+          />
+        </b-form-group>
+      </b-col>
```

Comment:
If you recall when we discussed this a bit back; I would omit what block the element came from.
I would just push all available elements in an array and show those also.
We can use the block label to prefix the element for nicer UX

Reply:
Ack